src/pybind/mgr/dashboard_v2/frontend/src/app/core/not-found/not-found.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1>Sorry, we could not find what you were looking for</h1>
+    <h2>Instead, enjoy some cephalopods!!!</h2>
```

Comment:
Per our discussion, let's stick to displaying the static image here and replace it with a new image once a new Ceph release has been made. I suggest to simply remove the `<h2>` then.